core/src/main/resources/db/changelog/changesets/2025_01_14_survey_response_metadata.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: "survey_response_metadata"
```

Comment:
there aren't any referral sources captured in production yet so i'm not going to bother writing a backfill for this. i'll check again post-release to be sure that nothing was created in the meantime